src/core/labeling/qgslabeling.h

```diff
@@ -30,6 +31,8 @@
  */
 class CORE_EXPORT QgsLabeling
 {
+    Q_GADGET
```

Comment:
What is the Q_GADGET for?

Reply:
To be able to use the Q_ENUM macro

Reply:
Better approach is to move the new enum to the Qgis class -- that's the general direction we're heading now

Reply:
Ok i can move it there